bundles/specmate-model-generation/src/com/specmate/modelgeneration/internal/cira/CiraClient.java

```diff
@@ -0,0 +1,69 @@
+package com.specmate.modelgeneration.internal.cira;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.ws.rs.core.Response.Status;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.osgi.service.log.LoggerFactory;
+
+import com.specmate.common.exception.SpecmateException;
+import com.specmate.common.exception.SpecmateInternalException;
+import com.specmate.model.administration.ErrorCode;
+import com.specmate.rest.RestClient;
+import com.specmate.rest.RestResult;
+
+/**
+ * Class to facilitate interacting with a Cira backend for classification and
+ * labeling (see https://github.com/JulianFrattini/cira-app)
+ */
+public class CiraClient {
+
+	private RestClient client;
+
+	public CiraClient(String ciraUrl, LoggerFactory loggerFactory) {
+		client = new RestClient(ciraUrl, 10, loggerFactory.getLogger(RestClient.class));
+	}
+
+	public boolean isCausal(String sentence) throws SpecmateException {
+		sentence = sentence.trim();
+		JSONObject object = new JSONObject();
+		object.put("sentence", sentence);
+		object.put("language", "en");
+		RestResult<JSONObject> result = client.post("/api/classify", object);

```

Comment:
Using an absolute path in the URL results in a user of the CiRA service being forced to make CiRA available under an extra IP or domain. We should use relative paths here.